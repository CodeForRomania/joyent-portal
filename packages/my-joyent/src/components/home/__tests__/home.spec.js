/**
 * @jest-environment jsdom
 */

import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Router } from '@mocks/';
import Home from '../home';

it('renders <Home /> without throwing', () => {
  const tree = renderer
    .create(
      <Router>
        <Home />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
