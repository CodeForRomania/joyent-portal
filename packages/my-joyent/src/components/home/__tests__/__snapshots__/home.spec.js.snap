// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Home /> without throwing 1`] = `
.c13 {
  background-color: transparent;
  text-decoration-skip: objects;
  color: ;
}

.c16 {
  font-family: sans-serif;
  font-size: 100%;
  line-height: 1.15;
  margin: 0;
  overflow: visible;
  text-transform: none;
  -webkit-appearance: button;
  -moz-appearance: button;
  appearance: button;
  min-width: 7.5rem;
  box-sizing: border-box;
  display: inline-block;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0;
  padding: 0.9375rem 1.125rem;
  position: relative;
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  font-size: 0.9375rem;
  text-align: center;
  font-style: normal;
  font-stretch: normal;
  line-height: normal;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  text-decoration: none;
  white-space: nowrap;
  vertical-align: middle;
  touch-action: manipulation;
  cursor: pointer;
  color: ;
  background-image: none;
  background-color: ;
  border-radius: 0.25rem;
  border: solid 0.0625rem;
}

.c16::-moz-focus-inner,
.c16[type='button']::-moz-focus-inner,
.c16[type='reset']::-moz-focus-inner,
.c16[type='submit']::-moz-focus-inner {
  border-style: none;
  padding: 0;
}

.c16:-moz-focusring,
.c16[type='button']:-moz-focusring,
.c16[type='reset']:-moz-focusring,
.c16[type='submit']:-moz-focusring {
  outline: 0.0625rem dotted ButtonText;
}

.c16:focus {
  outline: 0;
  text-decoration: none;
  background-color: ;
  border-color: ;
}

.c16:hover {
  background-color: ;
  border: solid 0.0625rem;
}

.c16:active,
.c16:active:hover,
.c16:active:focus {
  background-image: none;
  outline: 0;
  background-color: ;
  border-color: ;
}

.c16[disabled] {
  cursor: not-allowed;
  pointer-events: none;
}

.c16 + button {
  margin-left: 1.25rem;
}

.c10 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 0.9375rem;
}

.c4 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 1.5rem;
  font-weight: 400;
  color: ;
  margin: 1.25rem 0 1.125rem 0;
  display: inline-block;
}

.c5 {
  border: solid;
  border-width: 0 0.125rem 0.125rem 0;
  display: inline-block;
  padding: 0.125rem;
  -webkit-transform: rotate(-45deg);
  -ms-transform: rotate(-45deg);
  transform: rotate(-45deg);
  margin: 0.1875rem 0.625rem 0.1875rem 0.625rem;
}

.c0 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 1 auto;
  -ms-flex: 0 1 auto;
  flex: 0 1 auto;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin-right: -0.5rem;
  margin-left: -0.5rem;
}

.c12 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  line-height: 1.5rem;
  font-size: 0.9375rem;
}

.c15 {
  background-color: #FFFFFF;
  padding: 0.75rem;
  min-width: auto;
  border: none;
  box-shadow: none;
}

.c15:hover,
.c15:focus,
.c15:active,
.c15:active:hover,
.c15:active:focus {
  background-color: #FFFFFF;
  border: none;
  box-shadow: none;
}

.c6 {
  position: relative;
  margin-bottom: 0.75rem;
  background-color: ;
  box-shadow: 0 0.125rem 0 0 rgba(0,0,0,0.05);
  border: 0.0625rem solid;
}

.c7 {
  position: absolute;
  top: 0;
  left: 0;
  width: 2.25rem;
  height: 100%;
  background-color: ;
}

.c8 {
  padding: 0.75rem 0 0.84375rem 0;
}

.c9 {
  margin: 0 3.375rem 0.09375rem 3.375rem;
  font-weight: 600;
}

.c11 {
  margin: 0.09375rem 3.375rem 0 3.375rem;
}

.c14 {
  position: absolute;
  right: 0.1875rem;
  top: 0.1875rem;
}

.c2 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  display: inline-block;
  font-size: 0.9375rem;
  line-height: 1.6;
  margin-right: 1.4375rem;
}

.c3 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  color: ;
  text-decoration: none;
  cursor: pointer;
}

.c3.active {
  color: ;
  cursor: default;
}

.c1 {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

<main>
  .c0 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 1 auto;
  -ms-flex: 0 1 auto;
  flex: 0 1 auto;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin-right: -0.5rem;
  margin-left: -0.5rem;
}

.c2 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  display: inline-block;
  font-size: 0.9375rem;
  line-height: 1.6;
  margin-right: 1.4375rem;
}

.c3 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  color: ;
  text-decoration: none;
  cursor: pointer;
}

.c3.active {
  color: ;
  cursor: default;
}

.c1 {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

<div
    className="c0"
  >
    .c1 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  display: inline-block;
  font-size: 0.9375rem;
  line-height: 1.6;
  margin-right: 1.4375rem;
}

.c2 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  color: ;
  text-decoration: none;
  cursor: pointer;
}

.c2.active {
  color: ;
  cursor: default;
}

.c0 {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

<ul
      className="c0"
    >
      .c0 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  display: inline-block;
  font-size: 0.9375rem;
  line-height: 1.6;
  margin-right: 1.4375rem;
}

.c1 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  color: ;
  text-decoration: none;
  cursor: pointer;
}

.c1.active {
  color: ;
  cursor: default;
}

<li
        className="c0"
      >
        .c0 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  color: ;
  text-decoration: none;
  cursor: pointer;
}

.c0.active {
  color: ;
  cursor: default;
}

<a
          aria-current="true"
          className="active c0 active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Instances
        </a>
      </li>
      .c0 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  display: inline-block;
  font-size: 0.9375rem;
  line-height: 1.6;
  margin-right: 1.4375rem;
}

<li
        className="c0"
      >
        Custom images
      </li>
      .c0 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  display: inline-block;
  font-size: 0.9375rem;
  line-height: 1.6;
  margin-right: 1.4375rem;
}

<li
        className="c0"
      >
        Docker images
      </li>
      .c0 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  display: inline-block;
  font-size: 0.9375rem;
  line-height: 1.6;
  margin-right: 1.4375rem;
}

<li
        className="c0"
      >
        Docker registries
      </li>
    </ul>
  </div>
  .c1 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 1.5rem;
  font-weight: 400;
  color: ;
  margin: 1.25rem 0 1.125rem 0;
  display: inline-block;
}

.c2 {
  border: solid;
  border-width: 0 0.125rem 0.125rem 0;
  display: inline-block;
  padding: 0.125rem;
  -webkit-transform: rotate(-45deg);
  -ms-transform: rotate(-45deg);
  transform: rotate(-45deg);
  margin: 0.1875rem 0.625rem 0.1875rem 0.625rem;
}

.c0 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 1 auto;
  -ms-flex: 0 1 auto;
  flex: 0 1 auto;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin-right: -0.5rem;
  margin-left: -0.5rem;
}

<div
    className="c0"
    name="breadcrum"
  >
    .c0 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 1.5rem;
  font-weight: 400;
  color: ;
  margin: 1.25rem 0 1.125rem 0;
  display: inline-block;
}

.c1 {
  border: solid;
  border-width: 0 0.125rem 0.125rem 0;
  display: inline-block;
  padding: 0.125rem;
  -webkit-transform: rotate(-45deg);
  -ms-transform: rotate(-45deg);
  transform: rotate(-45deg);
  margin: 0.1875rem 0.625rem 0.1875rem 0.625rem;
}

<div>
      .c0 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 1.5rem;
  font-weight: 400;
  color: ;
  margin: 1.25rem 0 1.125rem 0;
  display: inline-block;
}

<h2
        className="c0"
        name="breadcrum-item"
      >
        Instances
      </h2>
      .c0 {
  border: solid;
  border-width: 0 0.125rem 0.125rem 0;
  display: inline-block;
  padding: 0.125rem;
  -webkit-transform: rotate(-45deg);
  -ms-transform: rotate(-45deg);
  transform: rotate(-45deg);
  margin: 0.1875rem 0.625rem 0.1875rem 0.625rem;
}

<div
        className="c0"
      />
    </div>
    .c0 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 1.5rem;
  font-weight: 400;
  color: ;
  margin: 1.25rem 0 1.125rem 0;
  display: inline-block;
}

.c1 {
  border: solid;
  border-width: 0 0.125rem 0.125rem 0;
  display: inline-block;
  padding: 0.125rem;
  -webkit-transform: rotate(-45deg);
  -ms-transform: rotate(-45deg);
  transform: rotate(-45deg);
  margin: 0.1875rem 0.625rem 0.1875rem 0.625rem;
}

<div>
      .c0 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 1.5rem;
  font-weight: 400;
  color: ;
  margin: 1.25rem 0 1.125rem 0;
  display: inline-block;
}

<h2
        className="c0"
        name="breadcrum-item"
      >
        Create Instance
      </h2>
      .c0 {
  border: solid;
  border-width: 0 0.125rem 0.125rem 0;
  display: inline-block;
  padding: 0.125rem;
  -webkit-transform: rotate(-45deg);
  -ms-transform: rotate(-45deg);
  transform: rotate(-45deg);
  margin: 0.1875rem 0.625rem 0.1875rem 0.625rem;
}

<div
        className="c0"
      />
    </div>
  </div>
  .c8 {
  background-color: transparent;
  text-decoration-skip: objects;
  color: ;
}

.c11 {
  font-family: sans-serif;
  font-size: 100%;
  line-height: 1.15;
  margin: 0;
  overflow: visible;
  text-transform: none;
  -webkit-appearance: button;
  -moz-appearance: button;
  appearance: button;
  min-width: 7.5rem;
  box-sizing: border-box;
  display: inline-block;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0;
  padding: 0.9375rem 1.125rem;
  position: relative;
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  font-size: 0.9375rem;
  text-align: center;
  font-style: normal;
  font-stretch: normal;
  line-height: normal;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  text-decoration: none;
  white-space: nowrap;
  vertical-align: middle;
  touch-action: manipulation;
  cursor: pointer;
  color: ;
  background-image: none;
  background-color: ;
  border-radius: 0.25rem;
  border: solid 0.0625rem;
}

.c11::-moz-focus-inner,
.c11[type='button']::-moz-focus-inner,
.c11[type='reset']::-moz-focus-inner,
.c11[type='submit']::-moz-focus-inner {
  border-style: none;
  padding: 0;
}

.c11:-moz-focusring,
.c11[type='button']:-moz-focusring,
.c11[type='reset']:-moz-focusring,
.c11[type='submit']:-moz-focusring {
  outline: 0.0625rem dotted ButtonText;
}

.c11:focus {
  outline: 0;
  text-decoration: none;
  background-color: ;
  border-color: ;
}

.c11:hover {
  background-color: ;
  border: solid 0.0625rem;
}

.c11:active,
.c11:active:hover,
.c11:active:focus {
  background-image: none;
  outline: 0;
  background-color: ;
  border-color: ;
}

.c11[disabled] {
  cursor: not-allowed;
  pointer-events: none;
}

.c11 + button {
  margin-left: 1.25rem;
}

.c5 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 0.9375rem;
}

.c0 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 1 auto;
  -ms-flex: 0 1 auto;
  flex: 0 1 auto;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin-right: -0.5rem;
  margin-left: -0.5rem;
}

.c7 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  line-height: 1.5rem;
  font-size: 0.9375rem;
}

.c10 {
  background-color: #FFFFFF;
  padding: 0.75rem;
  min-width: auto;
  border: none;
  box-shadow: none;
}

.c10:hover,
.c10:focus,
.c10:active,
.c10:active:hover,
.c10:active:focus {
  background-color: #FFFFFF;
  border: none;
  box-shadow: none;
}

.c1 {
  position: relative;
  margin-bottom: 0.75rem;
  background-color: ;
  box-shadow: 0 0.125rem 0 0 rgba(0,0,0,0.05);
  border: 0.0625rem solid;
}

.c2 {
  position: absolute;
  top: 0;
  left: 0;
  width: 2.25rem;
  height: 100%;
  background-color: ;
}

.c3 {
  padding: 0.75rem 0 0.84375rem 0;
}

.c4 {
  margin: 0 3.375rem 0.09375rem 3.375rem;
  font-weight: 600;
}

.c6 {
  margin: 0.09375rem 3.375rem 0 3.375rem;
}

.c9 {
  position: absolute;
  right: 0.1875rem;
  top: 0.1875rem;
}

<div
    className="c0"
  >
    .c7 {
  background-color: transparent;
  text-decoration-skip: objects;
  color: ;
}

.c10 {
  font-family: sans-serif;
  font-size: 100%;
  line-height: 1.15;
  margin: 0;
  overflow: visible;
  text-transform: none;
  -webkit-appearance: button;
  -moz-appearance: button;
  appearance: button;
  min-width: 7.5rem;
  box-sizing: border-box;
  display: inline-block;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0;
  padding: 0.9375rem 1.125rem;
  position: relative;
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  font-size: 0.9375rem;
  text-align: center;
  font-style: normal;
  font-stretch: normal;
  line-height: normal;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  text-decoration: none;
  white-space: nowrap;
  vertical-align: middle;
  touch-action: manipulation;
  cursor: pointer;
  color: ;
  background-image: none;
  background-color: ;
  border-radius: 0.25rem;
  border: solid 0.0625rem;
}

.c10::-moz-focus-inner,
.c10[type='button']::-moz-focus-inner,
.c10[type='reset']::-moz-focus-inner,
.c10[type='submit']::-moz-focus-inner {
  border-style: none;
  padding: 0;
}

.c10:-moz-focusring,
.c10[type='button']:-moz-focusring,
.c10[type='reset']:-moz-focusring,
.c10[type='submit']:-moz-focusring {
  outline: 0.0625rem dotted ButtonText;
}

.c10:focus {
  outline: 0;
  text-decoration: none;
  background-color: ;
  border-color: ;
}

.c10:hover {
  background-color: ;
  border: solid 0.0625rem;
}

.c10:active,
.c10:active:hover,
.c10:active:focus {
  background-image: none;
  outline: 0;
  background-color: ;
  border-color: ;
}

.c10[disabled] {
  cursor: not-allowed;
  pointer-events: none;
}

.c10 + button {
  margin-left: 1.25rem;
}

.c4 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 0.9375rem;
}

.c6 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  line-height: 1.5rem;
  font-size: 0.9375rem;
}

.c9 {
  background-color: #FFFFFF;
  padding: 0.75rem;
  min-width: auto;
  border: none;
  box-shadow: none;
}

.c9:hover,
.c9:focus,
.c9:active,
.c9:active:hover,
.c9:active:focus {
  background-color: #FFFFFF;
  border: none;
  box-shadow: none;
}

.c0 {
  position: relative;
  margin-bottom: 0.75rem;
  background-color: ;
  box-shadow: 0 0.125rem 0 0 rgba(0,0,0,0.05);
  border: 0.0625rem solid;
}

.c1 {
  position: absolute;
  top: 0;
  left: 0;
  width: 2.25rem;
  height: 100%;
  background-color: ;
}

.c2 {
  padding: 0.75rem 0 0.84375rem 0;
}

.c3 {
  margin: 0 3.375rem 0.09375rem 3.375rem;
  font-weight: 600;
}

.c5 {
  margin: 0.09375rem 3.375rem 0 3.375rem;
}

.c8 {
  position: absolute;
  right: 0.1875rem;
  top: 0.1875rem;
}

<div
      className="c0"
    >
      .c0 {
  position: absolute;
  top: 0;
  left: 0;
  width: 2.25rem;
  height: 100%;
  background-color: ;
}

<div
        className="c0"
      />
      .c5 {
  background-color: transparent;
  text-decoration-skip: objects;
  color: ;
}

.c2 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 0.9375rem;
}

.c4 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  line-height: 1.5rem;
  font-size: 0.9375rem;
}

.c0 {
  padding: 0.75rem 0 0.84375rem 0;
}

.c1 {
  margin: 0 3.375rem 0.09375rem 3.375rem;
  font-weight: 600;
}

.c3 {
  margin: 0.09375rem 3.375rem 0 3.375rem;
}

<div
        className="c0"
      >
        .c1 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 500;
  font-size: 0.9375rem;
}

.c0 {
  margin: 0 3.375rem 0.09375rem 3.375rem;
  font-weight: 600;
}

<h3
          className="c0 c1"
        >
          Choosing deployement data center
        </h3>
        .c2 {
  background-color: transparent;
  text-decoration-skip: objects;
  color: ;
}

.c1 {
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  line-height: 1.5rem;
  font-size: 0.9375rem;
}

.c0 {
  margin: 0.09375rem 3.375rem 0 3.375rem;
}

<p
          className="c0 c1"
        >
          .c0 {
  background-color: transparent;
  text-decoration-skip: objects;
  color: ;
}

<p>
            Not all data centres have all configurations of instances available. Make sure that you choose the data center that suits your requirements. 
            .c0 {
  background-color: transparent;
  text-decoration-skip: objects;
  color: ;
}

<a
              className="c0"
              href="#"
            >
              Learn More
            </a>
          </p>
        </p>
      </div>
      .c2 {
  font-family: sans-serif;
  font-size: 100%;
  line-height: 1.15;
  margin: 0;
  overflow: visible;
  text-transform: none;
  -webkit-appearance: button;
  -moz-appearance: button;
  appearance: button;
  min-width: 7.5rem;
  box-sizing: border-box;
  display: inline-block;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0;
  padding: 0.9375rem 1.125rem;
  position: relative;
  font-family: "Libre Franklin",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica,sans-serif;
  font-weight: 400;
  font-size: 0.9375rem;
  text-align: center;
  font-style: normal;
  font-stretch: normal;
  line-height: normal;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  text-decoration: none;
  white-space: nowrap;
  vertical-align: middle;
  touch-action: manipulation;
  cursor: pointer;
  color: ;
  background-image: none;
  background-color: ;
  border-radius: 0.25rem;
  border: solid 0.0625rem;
}

.c2::-moz-focus-inner,
.c2[type='button']::-moz-focus-inner,
.c2[type='reset']::-moz-focus-inner,
.c2[type='submit']::-moz-focus-inner {
  border-style: none;
  padding: 0;
}

.c2:-moz-focusring,
.c2[type='button']:-moz-focusring,
.c2[type='reset']:-moz-focusring,
.c2[type='submit']:-moz-focusring {
  outline: 0.0625rem dotted ButtonText;
}

.c2:focus {
  outline: 0;
  text-decoration: none;
  background-color: ;
  border-color: ;
}

.c2:hover {
  background-color: ;
  border: solid 0.0625rem;
}

.c2:active,
.c2:active:hover,
.c2:active:focus {
  background-image: none;
  outline: 0;
  background-color: ;
  border-color: ;
}

.c2[disabled] {
  cursor: not-allowed;
  pointer-events: none;
}

.c2 + button {
  margin-left: 1.25rem;
}

.c1 {
  background-color: #FFFFFF;
  padding: 0.75rem;
  min-width: auto;
  border: none;
  box-shadow: none;
}

.c1:hover,
.c1:focus,
.c1:active,
.c1:active:hover,
.c1:active:focus {
  background-color: #FFFFFF;
  border: none;
  box-shadow: none;
}

.c0 {
  position: absolute;
  right: 0.1875rem;
  top: 0.1875rem;
}

<button
        className="c0 c1 c2"
        onClick={[Function]}
      >
        <svg
          className=""
          height="12"
          viewBox="0 0 12 12"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>
            icon: close
          </title>
          <path
            d="M7.314 5.9l4.242-4.243L10.142.243 5.9 4.485 1.657.243.243 1.657l4.242 4.242-4.242 4.243 1.414 1.414 4.242-4.242 4.243 4.242 1.414-1.414L7.314 5.9z"
            fill="#494949"
            fillRule="evenodd"
          />
        </svg>
      </button>
    </div>
  </div>
</main>
`;
